### ISSUE TITLE
The in-process hosting model isn't supported for ASP.NET Core apps that target the .NET Framework.

### ISSUE THREAD

**Issue by jagwaran** [aspnet/AspNetCore.Docs#14435]
The in-process hosting model isn't supported for ASP.NET Core apps that target the .NET Framework.

What does "ASP.NET Core apps that target the .NET Framework" means? Also how do I host ASP.NET Core app in IIS with in-process hosting model?

**Comment by guardrex** [comment]
It means when you build an ASP.NET Core app that sets it's target framework to the .NET Framework. For example:

```xml
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>net461</TargetFramework>
  </PropertyGroup>

  ...

</Project>
```

For hosting and deployment guidance, see :point_right: https://docs.microsoft.com/aspnet/core/host-and-deploy/ and the topics under that node in the sidebar table of contents, especially https://docs.microsoft.com/aspnet/core/host-and-deploy/iis/. There's hosting model information in:

* https://docs.microsoft.com/aspnet/core/fundamentals/servers/
* https://docs.microsoft.com/aspnet/core/host-and-deploy/aspnet-core-module#hosting-models

For the .NET Core docs, you can get info over in a different doc set: https://docs.microsoft.com/dotnet/core/ ... you might want to access these lower-level docs first to get a foundation of basic concepts.

**Comment by guardrex** [comment]
Oh ... @jagwaran ... one more quick note. When you open a docs issue, use the feedback form for the page at the bottom of any topic. It adds metadata that helps us work the issue, and it automatically pings the author of the topic. No worries tho ... we'll get back to you either way ... it just makes things go a little smoother.

**Comment by jagwaran** [comment]
Thanks for the response @guardrex .

I installed ASP.NET Core Module (for IIS) and I have a simple ASP.NET Core 2.1 app with <TargetFramework>netcoreapp2.1</TargetFramework> in the vsproj project file

I published to IIS and I set the hostingModel="inprocess" in the web.config for aspNetCore.

But still, when I see the "Server:" in response header it says "Kestrel". This means, the app is running with OutOfProcess rt?

Also, is there any way problematically log/see what hostingModel app started with?
